volatility3/cli/__init__.py

```diff
@@ -458,10 +510,19 @@ def location_from_file(cls, filename: str) -> str:
         Returns:
             The URL for the location of the file
         """
-        vollog.debug(
-            f"{__name__}.location_from_file has been deprecated and moved to requirements.URIRequirement.location_from_file"
-        )
-        return requirements.URIRequirement.location_from_file(filename)
+        # We want to work in URLs, but we need to accept absolute and relative files (including on windows)
```

Comment:
All this code was moved to the `URIRequirement` class, I believe.  Best to leave it there, so that it's more part of the framework and not just the CLI.